packages/page-staking/src/useSortedTargets.ts

```diff
@@ -244,6 +247,28 @@ function extractBaseInfo (api: ApiPromise, allAccounts: string[], electedDerive:
   };
 }
 
+const b = (x: BN, api: ApiPromise): string => api.createType('Balance', x).toHuman();
+
+function getMinActiveThreshold (api: ApiPromise, stakers: [StorageKey<[u32, AccountId32]>, Codec][]) {
```

Comment:
this function seems correct to me, then my only guess is that `stakers` is not what we expect.

Reply:
I think I understood my mistake. Will correct ASAP

Reply:
Apparently I was mistaken - code is as in `polkadot-scripts`. Im not sure why the active min stake of nominators is so low.